mobile/node.go

```diff
@@ -444,9 +444,6 @@ func newHTTPGateway(node *core.OpenBazaarNode, ctx commands.Context, authCookie
 		return nil, fmt.Errorf("newHTTPGateway: manet.Listen(%s) failed: %s", gatewayMaddr, err)
 	}
 
-	// We might have listened to /tcp/0 - let's see what we are listing on
-	gatewayMaddr = gwLis.Multiaddr()
```

Comment:
why is this going away?

Reply:
`gatewayMaddr` is never used after that assignment.